resources/assets/components/InboxItem/index.js

```diff
@@ -64,6 +64,7 @@ class InboxItem extends React.Component {
               {campaign['reportback_info']['noun']} {campaign['reportback_info']['verb']}
             </div>
           </article>
+          <a href="#" onClick={e => this.props.showHistory(post['id'], e)}>Edit | Show History</a>
```

Comment:
@sbsmith86 do you think this part is okay for now? Once it's merged I can work on linking it to the backend and this seems like it would be quick to change later